litecli/main.py

```diff
@@ -825,8 +825,15 @@ def get_last_query(self):
         return self.query_history[-1][0] if self.query_history else None
 
 
+def version_callback(ctx: click.Context, param: click.Parameter, value: bool) -> None:
+    if not value or ctx.resilient_parsing:
+        return
+    click.echo(f"Version: {__version__}", color=ctx.color)
+    ctx.exit()
+
+
 @click.command()
-@click.option("-V", "--version", is_flag=True, help="Output litecli's version.")
+@click.option("-V", "--version", callback=version_callback, expose_value=False, is_eager=True, is_flag=True, help="Show version.")
```

Comment:
`@click.version_option()` will probably do what you want.  It uses importlib and assumes that the top-level name is your project name.

Impl at https://github.com/pallets/click/blob/fde47b4b4f978f179b9dff34583cb2b99021f482/src/click/decorators.py#L422